apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -73,6 +73,12 @@
 
     private static boolean sAnalyticsPaused;
 
+    private static String initialStartType;
```

Comment:
static variables that are not compile time constants must be prefixed with `s`

Reply:
Yes, resolved in new commit.